packages/core/admin/ee/admin/pages/SettingsPage/pages/Users/ListPage/CreateAction/index.js

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { useIntl } from 'react-intl';
+import PropTypes from 'prop-types';
+import { Stack, Button, Tooltip, Icon } from '@strapi/design-system';
+import { Envelop, ExclamationMarkCircle } from '@strapi/icons';
+import { useLicenseLimits } from '../../../../../../hooks';
+
+const CreateAction = ({ onClick }) => {
+  const { formatMessage } = useIntl();
+  const licenseLimitInfos = useLicenseLimits();
+  const { licenseLimitStatus, permittedSeats } = licenseLimitInfos;
```

Comment:
I think this should become:

```suggestion
  const { license } = useLicenseLimits();
  const { licenseLimitStatus, permittedSeats } = licenseLimitInfos.data ?? {};
```

If you want to display the loading state (rather than rendering nothing) you could get it via:

```
const { isLoading } = license;
```

Reply:
Yes, correct. I haven't pushed any code yet since merging your branch, I'll get to it 😊